submission/submission-config/src/main/kotlin/ac/uk/ebi/biostd/common/properties/ApplicationProperties.kt

```diff
@@ -3,6 +3,9 @@ package ac.uk.ebi.biostd.common.properties
 import org.springframework.boot.context.properties.ConfigurationProperties
 import org.springframework.boot.context.properties.ConstructorBinding
 
+/**
+ * @param subBasePath prefix of submission relPath. Should be null unless backend is used for e2e testing.
```

Comment:
comment can be way better IMO.  `Should be null unless backend is used for e2e testing.` this does not explain what the parameter does but rather tell the reader what not to do. We do not want to tell the developers what to do, we want to explain so they are able to decide it well informed.  Also this comment is for humans so you may favor use relative path instead of relPath for example/ 

`@param subBasePath extra path added as a prefix of submission relative path, Allows to allocate submissions under diferent paths`

Reply:
thanks for the comment! yes is quite clear